src/utility/utility.jl

```diff
@@ -143,3 +131,24 @@ end
 function fuser(::Type{T}, V1::S, V2::S) where {T,S<:IndexSpace}
     return isomorphism(T, fuse(V1 ⊗ V2), V1 ⊗ V2)
 end
+
+# TODO: remove once TensorKit supports this
+_twist(t::AbstractTensorMap, i::Int) = _twist(BraidingStyle(sectortype(t)), t, i)
+_twist(::Bosonic, t::AbstractTensorMap, i::Int) = t
+function _twist(::NoBraiding, t::AbstractTensorMap, i::Int)
```

Comment:
Is this superseded by Jutho/TensorKit.jl#240 now? If not, a more descriptive name might be good.